.travis.yml

```diff
@@ -4,44 +4,53 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
 
 env:
   matrix:
     
     - CONDA_PERL=5.20.3.1
+    - CONDA_PERL=5.22.0.1
+    - CONDA_PERL=5.22.2.1
```

Comment:
Sorry if I gave the impression that there is a `latest` pin. We need to hard-code

```yaml
    - perl 5.22.2.1
```

(I was lazy before and did not want to type the version number.)